app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_post
+
+  def index
+    comments = @post.comments
+    render json: { comments: }
+  end
+
+  def create
+    comment = @post.comments.new(comment_params)
+    comment.user = current_user
+
+    if comment.save
+      render json: { comment: }, status: :created
+    else
+      render json: { error: comment.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def set_post
+    @post = Post.find(params[:post_id])
+  end
+
+  def comment_params
+    params.require(:comment).permit(:content)
+  end
+end
```

Comment:
- [ ] @Team Great work on your application, however, you haven't added instructions in your readme file on how to log in from API testing software like Postman and be able to add a comment as previously requested.